diff --git a/feeds/luci/modules/luci-base/luasrc/model/network.lua b/feeds/luci/modules/luci-base/luasrc/model/network.lua
index 906bd2c327..1ac45ec523 100644
--- a/feeds/luci/modules/luci-base/luasrc/model/network.lua
+++ b/feeds/luci/modules/luci-base/luasrc/model/network.lua
@@ -23,7 +23,7 @@ module "luci.model.network"
 
 IFACE_PATTERNS_VIRTUAL  = { }
 IFACE_PATTERNS_IGNORE   = { "^wmaster%d", "^wifi%d", "^hwsim%d", "^imq%d", "^ifb%d", "^mon%.wlan%d", "^sit%d", "^gre%d", "^gretap%d", "^ip6gre%d", "^ip6tnl%d", "^tunl%d", "^lo$" }
-IFACE_PATTERNS_WIRELESS = { "^wlan%d", "^wl%d", "^ath%d", "^rausb%d", "^rai%d", "^ra%d", "^wdsi%d", "^wds%d", "^apclii%d", "^apcli%d", "^apcliusb%d", "^%w+%.network%d" }
+IFACE_PATTERNS_WIRELESS = { "^wlan%d", "^wl%d", "^ath%d", "^%w+%.network%d" }
 IFACE_ERRORS = {
        CONNECT_FAILED                  = lng.translate("Connection attempt failed"),
        INVALID_ADDRESS                 = lng.translate("IP address in invalid"),
