diff --git a/feeds.conf.default b/feeds.conf.default
index bff05ff178..32faac14f1 100644
--- a/feeds.conf.default
+++ b/feeds.conf.default
@@ -1,4 +1,4 @@
 src-git packages https://github.com/immortalwrt/packages.git;openwrt-18.06
-src-git luci https://github.com/padavanonly/luci.git;openwrt-18.06-k5.4
+src-git luci https://github.com/immortalwrt/luci.git;openwrt-18.06-k5.4
 src-git routing https://github.com/openwrt/routing.git
 src-git telephony https://github.com/openwrt/telephony.git
